File: app/src/main/java/com/yawar/memo/model/ChatRoomRespone.java
package com.yawar.memo.model;

import java.util.ArrayList;
import java.util.List;

public class ChatRoomRespone {
    private ArrayList<ChatRoomModel> data;

    public ChatRoomRespone(ArrayList<ChatRoomModel> items) {
        this.data = items;
    }

    public ArrayList<ChatRoomModel> getData() {
        return data;
    }

    public void setData(ArrayList<ChatRoomModel> data) {
        this.data = data;
    }
}

File: app/src/main/java/com/yawar/memo/model/ChatRoomRespone.kt
package com.yawar.memo.model

import com.squareup.moshi.Json
import com.yawar.memo.network.networkModel.ChatRoomModelDto
import java.util.*

data class ChatRoomRespone(

    @Json(name = "data")
    var data: ArrayList<ChatRoomModelDto?>? = null,

    )
